ft-logic/src/lib.rs

```diff
@@ -75,6 +75,35 @@ impl FTLogic {
                         self.approve(transaction_hash, account, &approved_account, amount)
                             .await;
                     }
+                    Action::PermitUnsigned {
+                        owner_account: _,
+                        approved_account: _,
+                        amount: _,
+                    } => {
+                        // TODO(m.strizhak) Where to report it?
+                        reply_err();
+                    }
+                    Action::Permit {
+                        owner_account,
+                        approved_account,
+                        amount,
+                        sign,
+                    } => {
+                        let payload = Action::PermitUnsigned {
```

Comment:
`Action` has a meaning of the messages the `logic` contract can accept, so it's better to take `PermitInsigned` out.

Reply:
Done